report/test/renderer/performance-category-renderer-test.js

```diff
@@ -160,7 +160,7 @@ describe('PerfCategoryRenderer', () => {
     const oppElements = [...categoryDOM.querySelectorAll('.lh-audit--load-opportunity')];
     expect(oppElements.map(e => e.id).sort()).toEqual(oppAudits.map(a => a.id).sort());
     expect(oppElements.length).toBeGreaterThan(0);
-    expect(oppElements.length).toMatchInlineSnapshot('7');
+    expect(oppElements.length).toMatchInlineSnapshot('6');
```

Comment:
What was the change here?

Reply:
I'm sure it was just image aspect ratio or responsive images no longer failing according to the artifacts.